.github/workflows/test-integrations.yml

```diff
@@ -425,7 +425,7 @@ jobs:
               `go list ./... | grep -v upgrade` \
               --target-image ${{ env.CONSUL_LATEST_IMAGE_NAME }} \
               --target-version local \
-              --latest-image docker.mirror.hashicorp.services/${{ env.CONSUL_LATEST_IMAGE_NAME }} \
```

Comment:
Why change this value?

Reply:
apparently we stopped publishing to `docker.mirror.hashicorp.services` with 1.16